jsonnet/dashboards/apps/loki.libsonnet

```diff
@@ -0,0 +1,246 @@
+local grafana = import 'grafonnet/grafana.libsonnet';
```

Comment:
Please add license at the beggining of file and format code like in other dasboards, for example [cadvisor](https://github.com/dNationCloud/kubernetes-monitoring/blob/main/jsonnet/dashboards/apps/cadvisor.libsonnet).

Also run linter and formatter from [Makefile](https://github.com/dNationCloud/kubernetes-monitoring/blob/main/Makefile) before each commit:
```
make jsonnet-fmt
make jsonnet-lint
```